reference/v8.json

```diff
@@ -1440,11 +1585,15 @@
       "function": "piecewise-constant",
       "zoom-function": true,
       "property-function": true,
-      "values": [
-        "map",
-        "viewport"
-      ],
-      "doc": "Control whether the translation is relative to the map (north) or viewport (screen)",
+      "values": {
+        "map": {
+            "doc": "The fill is translated relative to the map (north)."
```

Comment:
What does “north” mean here? Does it mean that a higher y value moves the fill’s origin to the north?